docs/INTEGRATION.md

```diff
@@ -79,7 +79,27 @@ Add the following job to your project's `.drone.yml` pipeline:
       - hadolint --version
       - hadolint Dockerfile
 ```
-
+## Jenkins declarative pipeline
+
+You can add a step during your CI proccess to lint and archive the output of hadolint
+
+```groovy
+stage ("lint dockerfile") {
+    agent {
+        docker {
+            image 'hadolint/hadolint:latest-debian'
+        }
+    }
+    steps {
+        sh 'hadolint dockerfiles/* | tee -a hadolint_lint.txt'
```

Comment:
In the Jenkins environment the checkstyle format could be digested by the [warnings-ng plugin](https://plugins.jenkins.io/warnings-ng).

```suggestion
        sh 'hadolint dockerfiles/* -f checkstyle | tee -a hadolint_lint.xml'
```

Reply:
looks like a good idea

Reply:
I was also experimenting with `hadolint -f json dockerfiles/*` then publishing to a slack channel using 
```
slackSend(
    message: "Dockerfile Error",
    attachments: hadolint_lint.json
)
```